packages/drivers/odsp-urlResolver/package.json

```diff
@@ -95,14 +95,17 @@
 		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
 	},
 	"dependencies": {
+		"@fluid-internal/client-utils": "workspace:~",
 		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/core-utils": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/odsp-driver": "workspace:~",
 		"@fluidframework/odsp-driver-definitions": "workspace:~"
 	},
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.13.3",
 		"@fluid-tools/build-cli": "^0.29.0",
+		"@fluid-tools/fluidapp-odsp-urlresolver-previous": "npm:@fluid-tools/fluidapp-odsp-urlresolver@2.0.0-internal.8.0.0",
```

Comment:
@chentong7 can you create an ADO item to track removing this? It should only be necessary right now because of the move from the removed package to the new one, but it should be removed in the next version bump, once we have a published "previous" for odsp-urlresolver, right?

Reply:
Have we verified that this works? To my knowledge, the type-tests only have awareness of the current package name, and look for `<package-name>-previous`. So I don't think they will ever see this dependency (worth double checking, but I'm pretty sure this is the case). In which case I'm not sure we get much benefit from adding this dep and moving over the type-tests from the old package.

FWIW, I don't think maintaining API compatibility with the old package is even needed.

Reply:
To clarify, what I mean is: Adding this dependency and copying over the generated type-tests module from the old package will ensure that running typescript builds the file we copied over, but no new changes will be detected for that module, since the type-test infrastructure doesn't know it exists. So, I'm not sure keeping it adds much benefit.

Reply:
Yes. I did verify it by `git clean -dxf` and then rebase and rebuilt from root. We do still have this package previous in pnpm module after that. Then should I exclude the type test to this class?

Reply:
Should I move FluidAppOdspUrlResolver class to existing file `urlResolver.ts` and add type test into this file?

Reply:
Updated